log/file.go

```diff
@@ -0,0 +1,157 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2020 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package log
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+// defaultLogfilePath is a default logfile path.
+const defaultLogfilePath = "/tmp/k6run.log"
+
+// fileHook is a hook to handle writing to local files.
+type fileHook struct {
+	fallbackLogger logrus.FieldLogger
+	ch             chan []byte
+	path           string
+	w              io.WriteCloser
+	levels         []logrus.Level
+}
+
+// FileHookFromConfigLine returns new fileHook hook.
+func FileHookFromConfigLine(
+	ctx context.Context, fallbackLogger logrus.FieldLogger, line string,
+) (logrus.Hook, error) {
+	hook := &fileHook{
+		fallbackLogger: fallbackLogger,
+		ch:             make(chan []byte),
+		path:           defaultLogfilePath,
```

Comment:
I don't think this needs a defaultLogFilePath especially as it will just log somewhere on hte system. Additionally the current one is Posix specific and likely will have problems on windows.

I do propose that for a first release we :
1. don't have a default one
2. if we decide to have one, it to actually be in the current directory, but I think this needs to be discussed more so let's open an issue instead ?

Reply:
I agree with your proposal

Reply:
I got rid of the default log file path